src/validation.cpp

```diff
@@ -1139,7 +1147,7 @@ bool MemPoolAccept::Finalize(const ATMPArgs& args, Workspace& ws)
     if (!args.m_package_submission && !bypass_limits) {
         LimitMempoolSize(m_pool, m_active_chainstate.CoinsTip());
         if (!m_pool.exists(GenTxid::Txid(hash)))
-            return state.Invalid(TxValidationResult::TX_MEMPOOL_POLICY, "mempool full");
+            return state.Invalid(TxValidationResult::TX_SINGLE_FAILURE, "mempool full");
```

Comment:
Might make sense to add a comment to the effect that we don't want to reject fetching this same tx in a future package